core/src/main/java/com/linecorp/armeria/server/ServerHttp1ObjectEncoder.java

```diff
@@ -188,7 +185,7 @@ boolean isSentConnectionCloseHeader() {
 
     @Override
     public boolean isResponseHeadersSent(int id, int streamId) {
```

Comment:
Good point. HTTP/2 encoder returns true if headers are sent. On second though, it would make sense to use `lastResponseHeadersId` newly defined in `Http1ObjectEncoder` rather than changing the method name.